spec/fixtures/deterministic.rs

```diff
@@ -0,0 +1,85 @@
+//
```

Comment:
I included this file for:
1. Confirmation I'm writing code that would catch deterministic clock and random principals.
2. Ease of change for future updates.